pkg/authenticator/k8s/authenticator.go

```diff
@@ -423,3 +424,17 @@ func consumeInjectClientCertError(path string) string {
 
 	return string(content)
 }
+
+func logIPS(host string) {
+	ips, lookUpError := net.LookupIP(host)
+	if lookUpError != nil {
+		fmt.Fprintf(os.Stderr, "Could not get IPs: %v\n", lookUpError)
+		os.Exit(1)
```

Comment:
Dont we want to return error if error happens and dont exit here like other places in this file?